src/core/qgsprovidermetadata.h

```diff
@@ -385,6 +386,12 @@ class CORE_EXPORT QgsProviderMetadata : public QObject
       const QgsCoordinateReferenceSystem &crs,
       const QStringList &createOptions = QStringList() ) SIP_FACTORY;
 
+    /**
+     * Creates a new instance of EPT point cloud data provider
+     * \since QGIS 3.20
+     */
+    SIP_SKIP virtual QgsEptProvider *createEptDataProvider( const QString &uri, const QString &dataSourceType, const QgsDataProvider::ProviderOptions &options, QgsDataProvider::ReadFlags flags = QgsDataProvider::ReadFlags() ) SIP_FACTORY;
```

Comment:
what if QGIS is build with WITH_EPT=False?

Reply:
@PeterPetrik excellent point! Some extra features that are not too common are getting entrenched and complicate the build process, thanks for raising this.